wordpress-mobile/WordPress-Android
# Co-branding JP login screen with WP.com

Fixes https://github.com/Automattic/jetpack-mobile-roadmap/issues/96
PT: pcdRpT-58l-p2

This PR redesigns the Jetpack app’s welcome screen to include more references to the WordPress brand, such as colours and logo.

| Before | After |
| ------  | ----- |
|<img width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/80d17a21-da3a-424e-989e-99cd6a7fb1f3"/>|<img width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/ad3e8644-ee6a-4947-b7dd-be6919b682d6"/>|
|<img width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/9c90e9c6-8a09-431a-b003-f6ab0e810f6f"/>|<img width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/106f1ca4-bc87-4012-8eaf-8a3d4ff73fbd"/>|

-----

## To Test:

- Run the app and compare the UI with the designs on Figma: qk5s4q4mZsXknwhMWo2vIY-fi-10924_2971
- Test both light and dark modes
- Make sure you can still login to the app

-----

## Regression Notes

1. Potential unintended areas of impact

    - Login screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual tests

3. What automated tests I added (or what prevented me from doing so)

    - UI changes only

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
